ehitchhiking-ui/src/app/shared/components/menu/menu.component.ts

```diff
@@ -14,6 +14,8 @@ import {Router} from '@angular/router';
 export class MenuComponent implements OnInit {
 	opened: boolean;
 	buttonsArray = [];
+	DIALOG_WINDOW_WIDTH = '23rem';
```

Comment:
pls create a file modal-constants in shared/constants and set 
export const DEFUALT_MAT_DIALOG_CLASS = 'mat-dialog-no-padding';
export const MAT_DIALOG_WIDTH_SM = '23rem';
export const MAT_DIALOG_WIDTH_MD = '40rem';
export const MAT_DIALOG_WIDTH_LG = '55rem';

Reply:
done